### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by tacheometry**
- Big encouragement for using Flatpak, with warning regarding other installation methods
- Auto-format files
- Console snippets should use `console` instead of `sh`
- Contributing should be last on the list
- Accessible link

### CODE REVIEW

**Comment by tacheometry** [comment]
After discussion with @lunarlettuce, I'll just have two articles, "Installation", and "Alternate installation".

**Comment by Arzumify** [comment]
I must ask one question:

Why would you put Flatpak first?
Like I get it has its advantages, but it's slower and takes up more space. Can't we just make all options equal?

**Comment by Arzumify** [comment]
Or just make the warning less "Scary"
Perhaps just put (easier) next to Flatpak or something

**Comment by Noted-Jrelvas** [comment]
Flatpak is rock solid and has its own environment curated by us. Most issues users report with Vinegar are related to whatever configuration and packages they have on their host.

**Comment by apprehensions** [comment]
Issue at hand here is the docs being unclear about how to setup Wine. Wine itself needs to have a special patch to setup Roblox, and what is being done here is to disregard that piece of information and tell the use to use the Flatpak.

I believe installations from source and package require the user to read the minimum requirements, and have a small guide on how to setup WineRoot.

File: src/Configuration/index.md [hunk 1]
```diff
@@ -58,16 +58,16 @@ This section are the available options for the global, Studio or Player configur
...
 
 <br/>Refer to the following table for valid `gpu` values:
 
 | Value              | Effect                                                                                                                                                    |
 | ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `"prime-discrete"` | Runs Roblox with your discrete gpu.*                                                                                                                      |
-| `"integrated"`     | Runs Roblox with your integrated graphics.*                                                                                                               |
+| `"prime-discrete"` | Runs Roblox with your discrete gpu.\*                                                                                                                     |
+| `"integrated"`     | Runs Roblox with your integrated graphics.\*                                                                                                              |
```
Comment (apprehensions) [inline src/Configuration/index.md:70]: Remove the asterisks

File: src/Configuration/index.md [hunk 2]
```diff
@@ -58,16 +58,16 @@ This section are the available options for the global, Studio or Player configur
 | `renderer`         | selects the rendering engine to be used by Roblox via FFlags.                                    | `"D3D11"`          |
 | `discord_rpc`      | use Discord's rich presence alongside handling the BloxstrapRPC protocol.                        | Player: `true`     |
 | `forced_version`   | forces Vinegar to use a specific version, the release channel must be adjusted for the version.  | none               |
-| `auto_kill_prefix` | automatically kill the wineprefix after the process exits                      .                 | `true`             |
+| `auto_kill_prefix` | automatically kill the wineprefix after the process exits .                                      | `true`             |
```
Comment (apprehensions) [inline src/Configuration/index.md:61]: Remove the space between the dot

File: src/Installation/guides/package.md [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
+<div class="warning">
+
+**Unless you have a good reason not to use it, it is strongly recommended to [install Vinegar using Flatpak](../index.md).**
+
+Running Vinegar and Roblox without Flatpak is prone to issues such as an incorrect Wine build, missing libraries, or improper configuration/installation.
+
+Configuring things properly is possible, but involves effort and debugging if Roblox doesn't work out of the box. That's why using Flatpak is recommended; it sets up and configures everything automatically, so it has the best chance of working smoothly the first time without needing any adjustments.
+
+</div>
+
```
Comment (apprehensions) [inline src/Installation/guides/package.md:12]: Ask the user to read the minimum requirements

File: src/Installation/guides/snap.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
 
+<div class="warning">
+
+**Unless you have a good reason not to use it, it is strongly recommended to [install Vinegar using Flatpak](../index.md).**
+
+Running Vinegar and Roblox without Flatpak is prone to issues such as an incorrect Wine build, missing libraries, or improper configuration/installation.
+
+Configuring things properly is possible, but involves effort and debugging if Roblox doesn't work out of the box. That's why using Flatpak is recommended; it sets up and configures everything automatically, so it has the best chance of working smoothly the first time without needing any adjustments.
+
+</div>
```
Comment (apprehensions) [inline src/Installation/guides/snap.md:11]: Remove this, as snap will have a wine build that solves Roblox failing to run.

File: src/Installation/guides/source.md [hunk 5]
```diff
@@ -1,24 +1,34 @@
...
 
+<div class="warning">
+
+**Unless you have a good reason not to use it, it is strongly recommended to [install Vinegar using Flatpak](../index.md).**
+
+Running Vinegar and Roblox without Flatpak is prone to issues such as an incorrect Wine build, missing libraries, or improper configuration/installation.
+
+Configuring things properly is possible, but involves effort and debugging if Roblox doesn't work out of the box. That's why using Flatpak is recommended; it sets up and configures everything automatically, so it has the best chance of working smoothly the first time without needing any adjustments.
+
+</div>
```
Comment (apprehensions) [inline src/Installation/guides/source.md:11]: Ask the user to read the minimum requirements

File: src/Installation/guides/source.md [hunk 6]
```diff
@@ -1,24 +1,34 @@
...
+
 ## Installing Wine
+
 Before you can install Vinegar, you will have to install Wine first. You can use the guide for your distribution on the [wine wiki](https://wiki.winehq.org/Download). For instance, you can find guides for [Arch](https://wiki.archlinux.org/title/wine) and [Gentoo](https://wiki.gentoo.org/wiki/Wine). Additionally, you will need the GnuTLS library in-order for Roblox networking to function properly:
 
 - Arch: `lib32-gnutls`
 - Void: `gnutls-32bit` via `void-repo-multilib`
 - Gentoo: `net-libs/gnutls` via `ABI_X86="64 32"`, refer to [Gentoo Wiki: package.use](https://wiki.gentoo.org/wiki//etc/portage/package.use)
```
Comment (apprehensions) [inline src/Installation/guides/source.md:19]: GnuTLS is not required

File: src/Troubleshooting/winehq.md [hunk 7]
```diff
@@ -59,11 +60,12 @@ discord_rpc = false           # (X)
...
 ```
+
 Read over every option carefully and follow the instructions in the comments.
 
 **Critical**: This configuration file depends heavily on your Vinegar version! <u>Do **NOT** reuse it</u>, especially after updating Vinegar. Recheck this guide to ensure you have the latest version of the configuration.
 
-Once the file is ready, set the `WINEPREFIX` env var to a temporary location, then run Vinegar with the `-config (filepath)` flag to use the cfg you just created. 
+Once the file is ready, set the `WINEPREFIX` env var to a temporary location, then run Vinegar with the `-config (filepath)` flag to use the cfg you just created.
 
 The command should be similar to this if all steps were followed correctly: `WINEPREFIX=/tmp/report vinegar -config /home/jrelvas/Documents/winehq-report.toml player`.
```
Comment (apprehensions) [inline src/Troubleshooting/winehq.md:70]: Why was this removed?

**Review by apprehensions — CHANGES_REQUESTED** [review]

File: src/Troubleshooting/winehq.md (hunk 7 above)
Comment (Noted-Jrelvas) [reply src/Troubleshooting/winehq.md:70]: Yeah, this shouldn't be removed. I even went out of my way to make it not outdated.

Comment (tacheometry) [reply src/Troubleshooting/winehq.md:70]: It wasn't removed. GitHub doesn't show it properly. But I ran an auto formatter, and that's probably what triggered a change - such as changing the line ending.

File: src/Configuration/index.md (hunk 1 above)
Comment (apprehensions) [reply src/Configuration/index.md:70]: Yes, but just keep it as a footnote, as other descriptions dont have the same style.

**Comment by apprehensions** [comment]
SGTM
